backend/README.md

```diff
@@ -2,7 +2,9 @@
 
 ## Alembic database init
 
-Please edit `alembic.ini` and specify your database URI in the parameter `sqlalchemy.url = postgresql://<username>:<password>@<host>/<database_name>`.
+~~Please edit `alembic.ini` and specify your database URI in the parameter `sqlalchemy.url = postgresql://<username>:<password>@<host>/<database_name>`.~~
```

Comment:
Since the alembic upgrades now happen automatically this whole section could be removed.